titleToNumber.js

```diff
@@ -0,0 +1,41 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/excel-sheet-column-number/
+const titleToNumber = (columnTitle) => {
```

Comment:
I would put in a blank line in between **comment** and **function declaration** because this comment is meant to describe the whole file, not just this function (you can have multiple functions in this file to solve a particular task).